src/python/grpcio/grpc/framework/foundation/callable_util.py

```diff
@@ -39,7 +39,7 @@ class Kind(enum.Enum):
         """Identifies the general kind of the outcome of some call."""
 
         RETURNED = object()
-        RAISED = object()
+        RAISED = object()  # noqa: PIE796
```

Comment:
Doesn't seem like the value is important here. It's not used anywhere. Let's just use [auto](https://docs.python.org/3.9/library/enum.html#using-automatic-values). We can migrate it to [StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum) later.

```suggestion
        RETURNED = enum.auto()
        RAISED = enum.auto()
```